x-pack/test/security_functional/page_objects/detections/index.ts

```diff
@@ -0,0 +1,157 @@
+/*
```

Comment:
__issue:__ looks like this file should be placed somewhere in `x-pack/test/security_solution_*` folders, `x-pack/test/security_functional` is dedicated to Kibana Security Platform tests (`x-pack/plugins/security`).

Reply:
thank you for letting me know! file is moved to `x-pack/test/security_solution_ftr`